Source/ACE.Server/WorldObjects/Monster_Tick.cs

```diff
@@ -64,7 +67,9 @@ public virtual void Monster_Tick(double currentUnixTime)
                 if (IsAnimating)
                 {
                     //PhysicsObj.ShowPendingMotions();
+                    ServerPerformanceMonitor.ResumeEvent(ServerPerformanceMonitor.MonitorType.Landblock_Tick_Monster_Tick_update_object);
```

Comment:
the main update_object for monsters would be the one on Monster_Navigation line 259 (moving around the world)